src/Console/Configuration/OptionsResolverConfigurator.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Lmc\Steward\Console\Configuration;
+
+use Lmc\Steward\Selenium\CapabilitiesResolver;
+use Lmc\Steward\Selenium\CapabilitiesResolverInterface;
+use Symfony\Component\OptionsResolver\Exception\InvalidOptionsException;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Configure given OptionsResolver instance to values accepted by Steward
+ */
+class OptionsResolverConfigurator
+{
+    /**
+     * @param OptionsResolver $optionsResolver
+     * @return OptionsResolver
+     */
+    public function configure(OptionsResolver $optionsResolver)
+    {
+        $this->configureCapabilitiesResolverOption($optionsResolver);
+        $this->configureDirsOption($optionsResolver);
+
+        return $optionsResolver;
+    }
+
+    private function configureCapabilitiesResolverOption(OptionsResolver $optionsResolver)
+    {
+        $optionsResolver->setDefaults([
+            ConfigOptions::CAPABILITIES_RESOLVER => CapabilitiesResolver::class,
+        ]);
+
+        $optionsResolver->setAllowedValues(ConfigOptions::CAPABILITIES_RESOLVER, function ($value) {
+            // Note we throw an exception instead of returning false - to have more understandable exception message
+            if (!class_exists($value)) {
```

Comment:
`Assertion::classExists` ?
or is it because of specific OptionsResolver exception?